SignalServiceKit/src/Messages/DeviceSyncing/OWSSyncOpenGroupsMessage.h

```diff
@@ -0,0 +1,15 @@
+#import "OWSOutgoingSyncMessage.h"
```

Comment:
Better to put this with the rest of the Session specific code instead of mixing it with the Signal code